Upgrade for EE5

EpicVoyage opened the pull request:
I know this code is probably abandoned at this point. Since I have found the plugin useful, I will leave this updated version here anyway.

Commit:
Updated to support EE5

Commit:
Updated version number in README

Commit:
Made usage function static for new PHP versions.

Commit:
Cleaned up reference to ee super-object

pvledoux commented:
Hi Chris, thanks for your PR. I'm indeed not maintaining any EE plugins anymore. If you want to, I can reference your fork as the maintained version?

I'm going to merge anyway.

EpicVoyage commented:
We can make my fork the maintained version. I just updated an old site to EE5 and updated a handful of plugins with it. If anybody does come knocking then I will have an environment to test.
